### ISSUE TITLE
Checkrain fails to boot jailbreak

### ISSUE THREAD

**Issue by fadedos** [checkra1n/BugTracker#232]
**What device + iOS version are you on?**
iphone 10, ios 12.2.2

**What checkra1n version are you using?**
0.9.2
**What are the steps to reproduce the issue?**
1. enter DFU mode
2. load the jailbreak
3.
...

**What do you expect, and what is happening instead?**
I expect the iphone to boot jailbroken but instead it says "booting" and never does until minutes later and it's still unjailbroken.

**Any other info, error logs, screenshots, ...?**
on the checkrain app on the computer it displays no error message and even says "done" after the bar is full.

**Comment by kpwn** [comment]
12.2.2 is not supported.

**Comment by fadedos** [comment]
> 12.2.2 is not supported.

It was working perfectly before and I know it works for others. I don’t think the update matter as long as you have an a11 device or lower

**Comment by CRKatri** [comment]
Use 0.9.2
When apple logo comes up start holding volume up
It will boot
Then open checkra1n app on the phone and restore filesystem

**Comment by kpwn** [comment]
You typed 12.2.2, and you meant 13.2.2. It took me a while to realise 12.2.2 is not an actual firmware and it was probably just a typo, which is why I removed my reply. It's a valid issue and I'm looking into it.

**Comment by fadedos** [comment]
> You typed 12.2.2, and you meant 13.2.2. It took me a while to realise 12.2.2 is not an actual firmware and it was probably just a typo, which is why I removed my reply. It's a valid issue and I'm looking into it.

Oh damn my fault, I got a reply on another thread saying to launch in safe mode, reset on the checkra1n app, re-jailbreak and download Cydia again and it worked although all of my tweaks were uninstalled but it’s whatever at least I’m jailbroken again.

**Comment by perpelator** [comment]
@fadedos If you haven’t figured it out still, try this (if u haven’t already):  1. Plug in your device. 2: Click start and get into recovery mode via Checkra1n (don’t manually go into recovery mode via buttons yourself. Let checkra1n do it for you by pressing next). 3: Once your device is detected again, begin to start the DFU mode process. After get ready (which is 3 seconds), the next step will begin at 6 seconds. Do NOT begin until that hits 5 seconds. 4: Continue to do what it says at the normal amount of said time.

Personally I started at 6 seconds but then cut all the other steps short by 1 second, but it’ll work either way. I personally was using a Hackintosh via usb a 2.0 on MacOS 10.13.6. Cannot confirm this method works (but it should) for other versions. I tested this on both 0.9 and 0.9.1, so should carry over with the following versions until fixed. I was also getting the same issue with booting into stock but this fixed it for me. Happy Jailbreaking :)
